### PR TITLE
Fix benchmarks

### PR DESCRIPTION
**Pull request by G-D-Petrov**
#### Reference Issues/PRs
Fixes the benchmark tests as there are 2 problems that are being addressed:
- persistent QB benchmarks
    - before, these benchmarks were not writing anything, just reading, as everything was prewritten in a specific prefix
    - this made the benchmarks run faster, but they were fragile to changes in the region/bucket, and we had such a change in December which broke them
    - now these benchmarks do the writing during the setup to make sure that the data is there
-  basic operations benchmarks
    - there was a change that led to writing a lot of data, much more than we could handle (>50GB)
    - this was caused by testing read/writes on short, wide dfs against many symbols
    - we have agreed with Alex O, that we don't need to tests this case against many symbols
    - now we write a short, wide dfs to only one symbols and benchmark that
    
 P. S. : this PR also contains the disabling of code cov on the pytest, due to a seg fault there

### CODE REVIEW

**Comment by poodlewars** [comment]
You should rebase commit 3557fcb787eada3f0d1bcf1763dc1480bc0ab113 out shouldn't you?

File: python/benchmarks/common.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def generate_random_floats_dataframe(num_rows, num_cols):
     return pd.DataFrame(data, columns=columns)
 
 
-def generate_benchmark_df(n, freq="T", end_timestamp="1/1/2023"):
+def generate_benchmark_df(n, freq="min", end_timestamp="1/1/2023"):
```
Comment (poodlewars) [inline python/benchmarks/common.py:40]: This is the same, isn't it?

File: python/benchmarks/common.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 import numpy as np
 
 
-def generate_pseudo_random_dataframe(n, freq="S", end_timestamp="1/1/2023"):
+def generate_pseudo_random_dataframe(n, freq="s", end_timestamp="1/1/2023"):
```
Comment (poodlewars) [inline python/benchmarks/common.py:12]: What does changing "S" to "s" do?

File: python/benchmarks/common.py (hunk 1 above)
Comment (G-D-Petrov) [reply python/benchmarks/common.py:40]: Yes, but T is apparently deprecated, and we get warnings from pandas which are quite annoying.

File: python/benchmarks/common.py (hunk 2 above)
Comment (G-D-Petrov) [reply python/benchmarks/common.py:12]: "S" is apparently deprecated, and we should use "s"

File: python/benchmarks/basic_functions.py [hunk 3]
```diff
@@ -15,12 +16,12 @@ class BasicFunctions:
     number = 5
     timeout = 6000
 
-    params = ([100_000, 150_000], [500, 1000])
+    params = ([10_000, 15_000], [500, 1000])
```
Comment (alexowens90) [inline python/benchmarks/basic_functions.py:19]: Why this change? IMO these row counts are already very low, and so not testing very representative use cases

Comment (G-D-Petrov) [reply python/benchmarks/basic_functions.py]: I just wanted to see what would be the time difference with the smaller rows.
The benchmarks runs around 20-25 min faster with the smaller values, which isn't that much faster (~140 vs ~160 min total run time).

I have brought the old values back.

**Review by alexowens90 — APPROVED** [review]
